ci-operator/step-registry/upi/deprovision/azurestack/upi-deprovision-azurestack-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+TENANT_ID=$(cat ${SHARED_DIR}/TENANT_ID)
+APP_ID=$(cat ${SHARED_DIR}/APP_ID)
+AAD_CLIENT_SECRET=$(cat ${SHARED_DIR}/AAD_CLIENT_SECRET)
+
+az cloud register \
+    -n PPE \
+    --endpoint-resource-manager "https://management.ppe3.stackpoc.com" \
+    --suffix-storage-endpoint "ppe3.stackpoc.com" 
```

Comment:
We don't want these values hardcoded into the steps. I'm ok with this for now, but it should be a priority to figure out how to parameterize this. It's really just a question of where we would store the value `ppe3.stackpoc.com`. Should that be an env var as part of the config or is it part of the cluster profile? Something else?

Reply:
Moved them to the ref.yaml. Should work.

Reply:
This value would be available in the install config if that is still kicking around in the SHARED_DIR